src/history/hs_verify.c

```diff
@@ -59,15 +56,6 @@ __hs_verify_id(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *ds_cbt, uint32_t this_
         if (btree_id != this_btree_id)
```

Comment:
Do we still need this check. Don't we get the return code as WT_NOTFOUND, if btree_id is not matching?

Reply:
Yes, we don't set the btree id in one of the caller.

Reply:
I agree - we can actually change this to an assert.

Reply:
It just walks the whole hs tree.

Reply:
Got it.